### PR TITLE
Second Attempt at SSCHA FC interface

### PR DESCRIPTION
**Pull request by dafolkner**
Here is a second attempt at a SSCHA fc interface for kALDo.

### CODE REVIEW

File: kaldo/observables/thirdorder.py [hunk 1]
```diff
@@ -168,7 +168,8 @@ def load(cls, folder, supercell=(1, 1, 1), format='sparse', third_energy_thresho
             replicated_atom_prime_file = str(folder) + '/' + replicated_filename
             atoms = ase.io.read(atom_prime_file)
             replicated_atoms = ase.io.read(replicated_atom_prime_file)
-            with open('FC3', 'r') as f:
+            file_name = str(folder) + '/' + 'FC3'
+            with open(file_name, 'r') as f:
                 lines = f.readlines()
                 f.close()
             c = 13.605693012183622 * (1.889725989 ** (3))
```
Comment (gbarbalinardo) [inline kaldo/observables/thirdorder.py:175]: What are these values?
`c = 13.605693012183622 * (1.889725989 ** (3))`

**Review by gbarbalinardo — COMMENTED** [review]
I've left a comment to address.
Did you have a chance to test it?

File: kaldo/observables/thirdorder.py (hunk 1 above)
Comment (gbarbalinardo) [reply kaldo/observables/thirdorder.py:175]: If you want to use Rydberg or other units, please import the ASE module.

Comment (ZKC19940412) [reply kaldo/observables/thirdorder.py:175]: The following site might be helpful: https://wiki.fysik.dtu.dk/ase/ase/units.html
Alternatively, scipy also has functions to handle units/constants: https://docs.scipy.org/doc/scipy/reference/constants.html

File: kaldo/observables/thirdorder.py [hunk 2]
```diff
@@ -168,10 +168,11 @@ def load(cls, folder, supercell=(1, 1, 1), format='sparse', third_energy_thresho
             replicated_atom_prime_file = str(folder) + '/' + replicated_filename
             atoms = ase.io.read(atom_prime_file)
             replicated_atoms = ase.io.read(replicated_atom_prime_file)
-            with open('FC3', 'r') as f:
+            file_name = str(folder) + '/' + 'FC3'
+            with open(file_name, 'r') as f:
                 lines = f.readlines()
                 f.close()
-            c = 13.605693012183622 * (1.889725989 ** (3))
+            c = units.Rydberg * ((1/units.Bohr) ** (3)) # conversion factor from Ry/bohr^3 to eV/A^3
```
Comment (gbarbalinardo) [inline kaldo/observables/thirdorder.py:175]: Minor, no need for `(value)`. It should be `value`, otherwise Python creates a tuple.

**Review by gbarbalinardo — APPROVED** [review]
LGTM

**Comment by gbarbalinardo** [comment]
@dafolkner Is this ready to be merged?

File: kaldo/observables/thirdorder.py [hunk 3]
```diff
@@ -194,7 +195,7 @@ def load(cls, folder, supercell=(1, 1, 1), format='sparse', third_energy_thresho
                 n_sc = np.prod(supercell)
                 pbc_conditions = replicated_atoms.get_pbc()
                 dim = len(pbc_conditions[pbc_conditions == True])
-                third_hiphive_file = str(folder) + '/THIRD'
+                third_hiphive_file = str(folder) + '/THIRD.sheng'
```
Comment (gbarbalinardo) [inline kaldo/observables/thirdorder.py:198]: is this line correct?

**Review by gbarbalinardo — APPROVED** [review]
Minor question on a line. Other than that it looks good.

**Comment by gbarbalinardo** [comment]
@dafolkner can we close this PR?

**Pull request closed without merging**